aix/SPECS/4.2.0/wazuh-agent-4.2.0-aix.spec

```
@@ -54,8 +53,9 @@ mkdir -p ${RPM_BUILD_ROOT}%{_init_scripts}
 mkdir -p ${RPM_BUILD_ROOT}%{_localstatedir}/.ssh
 
 # Copy the files into RPM_BUILD_ROOT directory
-install -m 0640 ossec-init.conf ${RPM_BUILD_ROOT}%{_sysconfdir}
-install -m 0750 src/init/ossec-hids-aix.init ${RPM_BUILD_ROOT}%{_init_scripts}/wazuh-agent
+sed "s/WAZUH_HOME_TMP/%{_localstatedir}/g" src/init/templates/ossec-hids-aix.init > src/init/templates/ossec-hids-aix.init.tmp
```

```suggestion
sed "s:WAZUH_HOME_TMP:%{_localstatedir}:g" src/init/templates/ossec-hids-aix.init > src/init/templates/ossec-hids-aix.init.tmp
```
If `%{_localstatedir}` contains the character `/` (and it will do it since it is a path), `sed` may fail because the expression becomes ambiguous. `sed` won't be able to detect where the expression begins and ends.

---

Fixed, thanks! https://github.com/wazuh/wazuh-packages/pull/609/commits/8c38e5e1e772d5be7657dc2645626ae3de3f8798